lib/languageNotifier.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
+import 'package:the_crew_companion/utils/appLocalizations.dart';
+
+class LanguageNotifier extends ChangeNotifier {
```

Comment:
ca vaut le coup de faire une class dédiée ? ou on pourrait faire AppLocalizations hériter de ChangeNotifier ? vu qu'on y stock que 1 fonction 🤔

Reply:
On éviterai aussi le côté "static" de AppLocalizations vu qu'on récupérerait l'instance avec `Provider.of<AppLocalisations>(context);` comme pour le themeNotifier ?

Reply:
D'accord pour le premier point et économiser une classe. Par contre, la partie statique est vraiment utile/pratique pour l'appel à translate.

Reply:
et si on conserve le translate en static ? il peut faire le provider.of.... pour récupérer l'instance, et faire ensuite la trad ? 

J'ai pas encore bien saisi les subtilités, mais je me dis que si les tutos disent de passer par le provider.of plutôt que des grosses classes static pour getter/setter la valeur, il doit y avoir une raison 😅

Reply:
Dans les tutoriels officiels, ils supposent que tu disposes du context dès que tu souhaites traduire quelque chose. Du coup, tu n'as pas besoin de faire un appel statique à translate.

Reply:
du coup le setLocale pourrait aussi être mis en static finalement ?

Reply:
J'ai pu merger `AppLocalizations` et `LanguageNotifier`. J'ai préféré laissé uniquement le `translate` en `static`. Il est vrai que c'est un plus simple d'utilisation. Je n'ai pas pu mettre `setLocale` en `static` à cause du `notifyListeners`.

Reply:
noice !